### PR TITLE
Add repository basics

### PR DESCRIPTION
**Pull request by yahavi**
* Add README.md file
* Add GitHub Actions tests and static analysis
* Fix static analysis issues.
* Add CLA assistant
* Add CONTRIBUTING.md
* Add release generator
* Add issues and pull requests templates
* Add Frogbot
* Cleanup and fix some tests

### CODE REVIEW

File: .github/workflows/frogbot.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: "Frogbot"
```
Comment (eyalbe4) [inline .github/workflows/frogbot.yml]: Should we update the Frogbot workflow to use the latest updatea?

File: lru/lru_test.go [hunk 2]
```diff
@@ -114,10 +114,10 @@ func TestPurge(t *testing.T) {
 }
 
 func TestExpiry(t *testing.T) {
-	c := New(3, WithExpiry(100*time.Millisecond))
+	c := New(3, WithExpiry(time.Second))
```
Comment (eyalbe4) [inline lru/lru_test.go:117]: Was this change intentional? 100 milliseconds != 1 second.
The same goes for the below changes - 
```
time.Sleep(500 * time.Millisecond)

time.Sleep(700 * time.Millisecond)
```

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: lru/lru_test.go (hunk 2 above)
Comment (yahavi) [reply lru/lru_test.go:117]: Yes. This is a time-based test.
The change to 1 second is to make sure it passed in 100% of the cases.
